pkg/utils/common.go

```diff
@@ -109,3 +117,44 @@ func SanitizeRepoUrl(url string) string {
 
 	return url
 }
+
+func CreateAppSecretName(targetName string, repoURL string) string {
```

Comment:
nit - both arguments are string so you can change the function to take (targetName, repoURL string)

Reply:
Since this is an exported utility function, we should have a function comment.